backend/databases/postgres.py

```diff
@@ -17,20 +21,44 @@ def __init__(self):
 
     @classmethod
     def get_instance(cls):
+        """
+        Get the instance of the database connection
+        """
         if cls._instance is None:
             cls._instance = cls().__create_engine()
         return cls._instance
 
     @staticmethod
     def __create_engine():
+        """
+        Create the database connection if there is no any existing connection
+        """
         try:
             return create_engine(
-                Secrets.POSTGRES_DB_URI,
-                pool_size=100,
-                max_overflow=20,
-                pool_timeout=30,
-                pool_recycle=3600
+                Secrets.POSTGRES_URI,
+                pool_size=Constants.POOL_SIZE,
+                max_overflow=Constants.MAX_OVERFLOW,
+                pool_timeout=Constants.POOL_TIMEOUT,
+                pool_recycle=Constants.POOL_RECYCLE,
```

Comment:
Change naming, should be PostgresConnectorConfig

Reply:
I think no need to put them as `PostgresConnectorConfig. POOL_SIZE `, etc. Just put in either the `Constants` class or the `Secrets` class.

The `Constants` class is used to store global settings or variables that can be accessed in our API. The `Secrets` class stores all sensitive settings.